libc-bottom-half/mman/mman.c

```diff
@@ -126,3 +138,36 @@ int munmap(void *addr, size_t length) {
     // Success!
     return 0;
 }
+
+int msync (void *addr, size_t length, int flags) {
```

Comment:
Doesn't this function need to be declared in a header somewhere?

Reply:
It is declared in `mman.h`.

Reply:
So what, the declaration was always there but the definition wasn't? Interesting!